map_gen/maps/crash_site/events.lua

```diff
@@ -245,7 +245,7 @@ local function do_pole(entity)
         return
     end
 
-    local supply_area_distance = entity.prototype.supply_area_distance
+    local supply_area_distance = entity.get_supply_area_distance(entity.quality)
```

Comment:
I think this needs to be.
```suggestion
    local supply_area_distance = entity.prototype.get_supply_area_distance(entity.quality)
```